kms.tf

```diff
@@ -7,6 +7,14 @@ data "aws_caller_identity" "kms" {
 }
 
 data "aws_iam_policy_document" "trail_kms" {
+  #checkov:skip=CKV_AWS_1
+  #checkov:skip=CKV_AWS_41
```

Comment:
This finding means "Ensure AWS access keys and secrets are not hard coded in infrastructure" and it's used only in provider, it has nothing to do with IAM, please remove this suppress finding.